rubix_admin/installer.py

```diff
@@ -18,13 +19,12 @@ def setup_parsers(cls, sub_parser):
                                                        help="Install from RPM")
         cls.install_parser.add_argument("-r", "--rpm", required=True,
                                         help="Path to RPM file")
-        cls.install_parser.add_argument("-a", "--rpm-args", default="",
+        cls.install_parser.add_argument("-a", "--rpm-args", default="--ignoreos",
                                         help="Arguments to rpm command")
         cls.install_parser.set_defaults(func=cls.install_cmd)
 
     @classmethod
     def install_cmd(cls, args):
-        logging.info("Installing %s" % args.rpm)
```

Comment:
Reason for removing this log ? If it is too verbose, please move it to debug level.